kubernetes/config.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Configmap
+metadata:
+  name: issue-sync
+type: Opaque
+data:
+  log-level:
+  jira-uri:
+  repo-name:
+  jira-project:
+  timeout:
+  period:
```

Comment:
That entry is in the secrets file, although I suppose it probably isn't a secret. I just thought it would be clearer to keep all of the auth together. I can move it over though.